ebml/EbmlString.h

```diff
@@ -63,7 +63,7 @@ class EBML_DLL_API EbmlString : public EbmlElement {
     EbmlString &SetValue(std::string const &NewValue);
     std::string GetValue() const;
 
-    void SetDefaultValue(std::string &);
+    void SetDefaultValue(std::string);
```

Comment:
Uhm… the more appropriate type would be `std::string const &`, which will prevent having to create a copy a lot of the time.

Reply:
Right. Both work. I wasn't sure if clang-tidy would throw https://clang.llvm.org/extra/clang-tidy/checks/modernize/pass-by-value.html so I did it anyways.